plugins/modules/pagerduty_alert.py

```diff
@@ -53,11 +55,37 @@
             - 'triggered'
             - 'acknowledged'
             - 'resolved'
-    api_key:
+    api_version:
         type: str
         description:
-            - The pagerduty API key (readonly access), generated on the pagerduty site.
-        required: true
+            - The Api version we want to use to run the module.
```

Comment:
Please make sure to fully capitalize abbreviations.
```suggestion
            - The API version we want to use to run the module.
```